src/default.frag.glsl

```diff
@@ -6,9 +6,9 @@
 
 uniform sampler2DArray textures;
 
-attribute int frag_textureid;
-attribute vec2 frag_texcoord;
+varying vec2 frag_texcoord;
+varying float frag_textureid;
```

Comment:
So I guess `attribute` and `varying` describe the type rather than being analogs to the more modern `in` and `out`. You pointed it out on Discord, but I'm putting it here for documentation's sake.

https://stackoverflow.com/questions/17537879/in-webgl-what-are-the-differences-between-an-attribute-a-uniform-and-a-varying